arcade/gui/widgets/__init__.py

```diff
@@ -759,7 +760,7 @@ def __init__(
             size_hint_min=size_hint_min,
             size_hint_max=size_hint_max,
         )
-        self.color = (randint(0, 255), randint(0, 255), randint(0, 255))
+        self.color: RGBOrA255 = (randint(0, 255), randint(0, 255), randint(0, 255))
```

Comment:
To my understanding, this should be upgraded to be use full alpha and `Color.from_iterable` or similar.

Reply:
How about we extract that to an issue and tackle in a different PR?  That way, we can keep this PR focused solely on enabling pyright while preserving existing behavior.

Reply:
Sounds good. :+1:

Reply:
Nevermind me, the change to use `RGBA255` was tiny, I made the change.